packages/react-native-otlp/android/dependencies.gradle

```diff
@@ -0,0 +1,14 @@
+// custom dependencies specific to Embrace and OpenTelemetry
+import groovy.json.JsonSlurper
+
+def packageJsonFile = file("../package.json")
+
+def jsonSlurper = new JsonSlurper()
+def packageJson = jsonSlurper.parse(packageJsonFile)
+
+dependencies {
+  api "io.embrace:embrace-android-sdk:$packageJson.embrace.androidVersion"
+
+  implementation(platform("io.opentelemetry:opentelemetry-bom:1.43.0"))
+  implementation("io.opentelemetry:opentelemetry-exporter-otlp-http-trace:1.14.0")
```

Comment:
on the iOS side GRPC was exposed as well, will that be needed here as a dependency?

Reply:
I think I've removed the GRPC dependency from the iOS side, I don't see any mention of it (neither in the dependencies nor in the copied/pasted files)

Reply:
ah ok nm then